recipes/polyglotdb/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set name = "polyglotdb" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 89d2b4a19595a7bc14d1fba124afc63454baa9678275cd093d091d0430fc36c2
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - pytest
+    - python >=3.6
+  run:
+    - conch_sounds
+    - influxdb
+    - librosa >=0.8.0,=0.8.*
+    - neo4j-python-driver >=4.3,=4.*
```

```suggestion
    - neo4j-python-driver >=4.3,==4.*
```
I think `~=` gets translated into that. You might need to double-check the others as well.

---

I tested both, it seems to be single "="